pkg/backend/apply.go

```diff
@@ -125,6 +125,17 @@ func PreviewThenPrompt(ctx context.Context, kind apitype.UpdateKind, stack Stack
 		return plan, changes, nil
 	}
 
+	infoPrefix := "\b" + op.Opts.Display.Color.Colorize(colors.SpecWarning+"info: "+colors.Reset)
```

Comment:
modified to use the `warning` yellow color as we use it to indicate that there's a new version of pulumi.